src/network/networkd-manager.h

```diff
@@ -96,7 +96,7 @@ void manager_dirty(Manager *m);
 
 Link* manager_find_uplink(Manager *m, Link *exclude);
 
-int manager_set_hostname(Manager *m, const char *hostname);
+int manager_set_hostname(Manager *m, const char *hostname, bool static_hostname);
```

Comment:
Not a big matter, but the name `bool static_hostname` is redundant, `bool static` or `bool is_static` is enough.